src/main/java/net/fabricmc/loom/task/AbstractRemapJarTask.java

```diff
@@ -103,6 +105,9 @@ public abstract class AbstractRemapJarTask extends Jar {
 	@Input
 	public abstract ListProperty<String> getAdditionalClientOnlyEntries();
 
+	@Input
+	public abstract Property<String> getClientOnlySourceSetName();
```

Comment:
Fyi this could also also be an `@Internal Property<SourceSet>` if you added a custom input property on `getClientOnlySourceSet().get().getName()`, but probably overkill